### PR TITLE
feat: proposal for funding use guidelines

### PR DESCRIPTION
**Pull request by wesleytodd**
We had a start of a discussion in the TC slack. I didn't want to do that in private since we seemed to dont have anyone fully opposed to our plans for financial support for the project.  This is just a start, we would appreciate any input (even strong disagreement) here so that we are sure the maintainers, contributors, and community are comfortable with our plans.

### CODE REVIEW

**Review by UlisesGascon — APPROVED** [review]

File: docs/adr/funding-guidelines.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+- @wesleytodd
+
+## Decision Owners
+- @expressjs/express-tc
+
+## Context
+
+We have an Open Collective but we originally decided to wait on setting guidelines for how we will spend the funds. Now that we have landed PRs to include the `funding` field in
+`package.json` and also a link in the footer of the website, we wanted to have some loose guidelines so contributors know a little about how we intend to use financial support for
+the project.
```
Comment (ctcpip) [inline docs/adr/funding-guidelines.md:24]:
```suggestion
We have an Open Collective but we originally decided to wait on setting guidelines for how we will spend the funds. Now that we have landed PRs to include the `funding` field in `package.json` and also a link in the footer of the website, we wanted to have some loose guidelines so contributors know a little about how we intend to use financial support for the project.
```

File: docs/adr/funding-guidelines.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+- Living expenses (food, shelter, medicine/healthcare, transportation, utilities)
+  - The requester need not specify any detail other than the amount being requested.
+- Travel (hotel and airfare) to events related to Express or the greater Node/JS ecosystem
+  - The requester should specify the event, location, and cost estimates for hotel and airfare.
+- Equipment and hardware (devices, screens, internet services)
+  - The requester should specify the type of equipment or service needed and how it relates to their work on Express.
+
+Requests will be reviewed at the next possible opportunity, privately during an Express TC meeting. The Express TC also reserves the right to ammend these guidelines via our
+standard decision making process.
```
Comment (ctcpip) [inline docs/adr/funding-guidelines.md:40]:
```suggestion
Requests will be reviewed at the next possible opportunity, privately during an Express TC meeting. The Express TC also reserves the right to amend these guidelines via our standard decision making process.
```

**Review by ctcpip — CHANGES_REQUESTED** [review]

**Comment by crandmck** [comment]
LGTM

**Review by ctcpip — APPROVED** [review]

**Review by ljharb — APPROVED** [review]
Also, make sure to enable the "sponsors" checkbox on all org repos, and set up a FUNDING.yml in the org's .github repo.

**Review by bjohansebas — APPROVED** [review]

File: docs/adr/funding-guidelines.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+## Decision
+
+Funds are not automatically distributed.
+
+TC members may request funds to be distributed for the following purposes:
+
+- Living expenses (food, shelter, medicine/healthcare, transportation, utilities)
+  - The requester need not specify any detail other than the amount being requested.
+- Travel (hotel and airfare) to events related to Express or the greater Node/JS ecosystem
```
Comment (bjohansebas) [inline docs/adr/funding-guidelines.md:35]: Shouldn't we specify that they must be of legal age according to their locality?

Comment (UlisesGascon) [reply docs/adr/funding-guidelines.md:35]: Great catch! Following the CPC example: https://github.com/openjs-foundation/cross-project-council/blob/a177a05a469ca101ac158d22aa4fb4b1af8725f1/community-fund/COMMUNITY_FUND_POLICY.md?plain=1#L9

Comment (UlisesGascon) [inline docs/adr/funding-guidelines.md:35]:
```suggestion
- Travel (hotel and airfare) to events related to Express or the greater Node/JS ecosystem. _Legal Requirement: at least 18 years old and recognized as adults in both the country of departure and the destination country, based on local laws (ref: [Age of Majority](https://en.wikipedia.org/wiki/Age_of_majority))_
```

Comment (UlisesGascon) [reply docs/adr/funding-guidelines.md:35]: Added a siggestion for this https://github.com/expressjs/discussions/pull/299/files#r1830828177

**Comment by UlisesGascon** [comment]
I added the age restriction (same as CPC) in [`c4211d3` (#299)](https://github.com/expressjs/discussions/pull/299/commits/c4211d387fad8045c1f4ca9a8ae519f209535592)

Given that we have a basic agreement, I will merge this PR in order to unblock the Express releases :+1: 

Note: We can always make more changes in the future to polish and extend the policy :)

**Pull request merged**